source/_integrations/eway.markdown

```
@@ -0,0 +1,171 @@
+eway:
+  mqtt_host: "your-mqtt-broker.com"
+  mqtt_port: 1883
+  mqtt_username: "your_username"
+  mqtt_password: "your_password"
+  device_id: "your_device_id"
+  device_sn: "your_device_serial"
+  device_model: "your_device_model"
+  scan_interval: 30
+```
+
+## MQTT Topics
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)